catalog/rest/catalog-rest-endpoint/pom.xml

```diff
@@ -149,6 +149,14 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>ddf.features</groupId>
```

Comment:
❓ Should we consider, either now or in a future enhancement, pulling all of these `test-utilities` out to [codice-test](https://github.com/codice/codice-test)?

Reply:
I wouldn't be against it. I've been trying to keep this feature strictly thirdparty dependencies to avoid circular dependencies in our dependency tree. Moving it over there would guarantee that going forward.  We'd have to move the `test-common` artifact over there as well though. It has a dependency on `platform-util` that we'd need to break to do that.  The `test-common` artifact is also still changing often, we may want to wait a bit to avoid releasing frequently.

Before we move the `test-common` over to that repo, I think it needs to be refactored so we have one way of configuring the test container rather than several. I'm all for this long term though.